src/pages/GardenAdmin/GardenAdmin.tsx

```diff
@@ -23,7 +23,7 @@ const GardensAdmin = (): JSX.Element => {
   }, [data, error])
 
   return (
-    <div>
+    <>
```

Comment:
Agree this shouldn't be a raw `div`, but not so sure it should be a fragment. For consistency's sake, if nothing else, I think our page components should be rooted at their own DOM object. Though I'm willing to change my opinion if there's a good argument I'm not aware of. How about a `Box` in this case?